.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature Request
+about: Suggest a new feature or enhancement to Nomad
+labels: “type/enhancement”
```

Comment:
Probably want to make sure you don't have smart quotes in the metadata here, just in case GitHub chokes on them. (Same for the others)

```suggestion
labels: "type/enhancement"
```

Reply:
_shakes fist at google docs_